pymc/variational/opvi.py

```diff
@@ -1127,28 +938,6 @@ def _new_initial(self, size, deterministic, more_replacements=None):
             initial = at.switch(deterministic, at.ones(shape, dtype) * dist_map, sample)
             return initial
 
-    @node_property
-    def symbolic_random(self):
-        """*Dev* - abstract node that takes `self.symbolic_initial` and creates
-        approximate posterior that is parametrized with `self.params_dict`.
-
-        Implementation should take in account `self.batched`. If `self.batched` is `True`, then
-        `self.symbolic_initial` is 3d tensor, else 2d
-
-        Returns
-        -------
-        tensor
-        """
-        raise NotImplementedError
-
-    @node_property
-    def symbolic_random2d(self):
```

Comment:
was only used for batched or local inference. Any occurrence usage of this property can be safely replaced with `symbolic_random`

Reply:
done